src/commands/utils_command/send_signed_transaction/mod.rs

```diff
@@ -64,21 +64,34 @@ impl Transaction {
                     break response;
                 }
                 Err(err) => {
-                    if let Some(serde_json::Value::String(data)) = &err.data {
-                        if data.contains("Timeout") {
-                            println!("Timeout error transaction.\nPlease wait. The next try to send this transaction is happening right now ...");
-                            continue;
+                    match &err.data {
+                        Some(serde_json::Value::String(data)) => {
+                            if data.contains("Timeout") {
```

Comment:
Do we have an error kind here?

Reply:
@volovyk-s We will refactor it once we migrate to https://github.com/near/near-jsonrpc-client-rs